### DISCUSSION TITLE
yetiforce portal request

### DISCUSSION THREAD

**Discussion by kassimg12** [YetiForceCompany/YetiForceCRM#15487] [General]
Hello, yetiforce families

is there a free version of yetiforce portal?
I found a web service standard documentation at https://doc.yetiforce.com/api/#/
is it possible to use it for the customer portal here https://github.com/YetiForceCompany/YetiForcePortal2

Thank you!

**Comment by mariuszkrzaczkowski** [comment]
There is no free version of the portal. It consumed a lot of time to make that's why currently it's a paid feature. 
Maybe it'll be free in the future, once we're reimbursed for the cost of developing it.

**Comment by kassimg12** [comment]
Thank you for replying.

I appreciate your dedication for the product. It is well developed in my
perspective.

I wish I can subscribe for the portal. However I couldn't facilitate
continuous payment option for subscription.

If there is an option for one time payment to purchase portal and premium
web service, please let me know and I will go for it.

Thank you!
